azurerm/resource_arm_app_service_certificate.go

```diff
@@ -0,0 +1,260 @@
+package azurerm
+
+import (
+	"encoding/base64"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/web/mgmt/2018-02-01/web"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/validate"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/tags"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourceArmAppServiceCertificate() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmAppServiceCertificateCreateUpdate,
+		Read:   resourceArmAppServiceCertificateRead,
+		Update: resourceArmAppServiceCertificateCreateUpdate,
+		Delete: resourceArmAppServiceCertificateDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validate.NoEmptyStrings,
+			},
+
+			"location": azure.SchemaLocation(),
+
+			"resource_group_name": azure.SchemaResourceGroupName(),
+
+			"friendly_name": {
```

Comment:
Could we move all computed values to the bottom of the schema. IE properties should go required->optional->computed

Reply:
Made an attempt to fix this, but still left `tags` at bottom.